src/Elcodi/Bundle/AttributeBundle/DependencyInjection/ElcodiAttributeExtension.php

```diff
@@ -77,15 +77,15 @@ protected function getConfigurationInstance()
     protected function getParametrizationValues(array $config)
     {
         return [
-            "elcodi.core.attribute.entity.attribute.class" => $config['mapping']['attribute']['class'],
-            "elcodi.core.attribute.entity.attribute.mapping_file" => $config['mapping']['attribute']['mapping_file'],
-            "elcodi.core.attribute.entity.attribute.manager" => $config['mapping']['attribute']['manager'],
-            "elcodi.core.attribute.entity.attribute.enabled" => $config['mapping']['attribute']['enabled'],
+            "elcodi.entity.attribute.class" => $config['mapping']['attribute']['class'],
```

Comment:
i agree too, but that is out of the scope of this PR, should we create another ticket for that? I am finding that we have to pipe PRs else this becomes a nightmare of work :+1: